### PR TITLE
WIP: how to create a basic strategy

### PR DESCRIPTION
**Pull request by drgorillamd**

### CODE REVIEW

File: strategies/how-to-write-a-basic-strategy.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+# How to write a basic strategy
+
+Every strategy is, at its core, the implementation of the `strategy(space, network, provider, addresses, options, snapshot)` function.
+
+A quick word on the function parameters:
+
+* `space`: the space (if applicable), based on the registered ENS associated with this strategy (see [Spaces](../spaces/))
+* `network`: the network id as an integer (1 for Ethereum)
```
Comment (bonustrack) [inline strategies/how-to-write-a-basic-strategy.md:12]:
```suggestion
* `network`: the network id as a string ("1" for Ethereum mainnet)
```

File: strategies/how-to-write-a-basic-strategy.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* `provider`: the network provider used by the web browser
+* `addresses`: the list of addresses of the voters being processed.
+* `options`: a collection of options that can be passed to strategy(...) - those have no default keys and can be used to pass any useful data
+* `snapshots`: the block height at which the data are queried by strategy(...) - if none is passed, 'latest' is used.&#x20;
```
Comment (bonustrack) [inline strategies/how-to-write-a-basic-strategy.md:16]:
```suggestion
* `snapshot`: the block height at which the data are queried by strategy(...) - if none is passed, 'latest' is used.&#x20;
```

File: strategies/how-to-write-a-basic-strategy.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+`Strategy(...)` returns a javascript object with every addresses from the addresses array as keys and the corresponding voting power as value (expressed in number of token owned by the address for `erc20-balance-of`, as a number of rocks owned for `has-rock`, etc).
```
Comment (bonustrack) [inline strategies/how-to-write-a-basic-strategy.md:18]:
```suggestion
`Strategy(...)` returns a JavaScript object with every addresses from the addresses array as keys and the corresponding voting power as value (expressed in number of token owned by the address for `erc20-balance-of`, as a number of rocks owned for `has-rock`, etc).
```

File: SUMMARY.md [hunk 4]
```diff
@@ -17,6 +17,7 @@
 * [Strategies](strategies/README.md)
   * [What is a strategy?](strategies/what-is-a-strategy.md)
   * [Create a new strategy](strategies/create.md)
+  * [How to write a basic strategy](strategies/how-to-write-a-basic-strategy.md)
```
Comment (bonustrack) [inline SUMMARY.md:20]: Isn't "Create a new strategy" having the same purpose than "How to write a basic strategy"? Shouldnt this be merged together?

Comment (drgorillamd) [reply SUMMARY.md:20]: Indeed, kinda hesitated on this and ended up thinking it would maybe be too big... Do you want me to merge them?

**Pull request closed without merging**